install/index.adoc

```diff
@@ -290,6 +290,11 @@ container method is used on that host. Either installation type provides the
 same functionality for the cluster, but the operating system you use determines
 how you manage services and host updates.
 
+[NOTE]
+====
+As of {product-title} 3.10, the containzerized installation method is no longer supported on Red Hat Enterprise Linux systems.
```

Comment:
s/Red Hat Enterprise Linux/{op-system-base-full} systems

The {op-system-base-full} and other variables are defined in the modules/common-attributes.adoc file

Reply:
Oh! Since this is 3.x docs, that variable may not exist. If it doesn't, disregard :smile: